metadata-ingestion/src/datahub/ingestion/source/sql/bigquery.py

```diff
@@ -398,28 +397,42 @@ def _get_bigquery_log_entries(
         template: str,
     ) -> Union[Iterable[AuditLogEntry], Iterable[BigQueryAuditMetadata]]:
         # Add a buffer to start and end time to account for delays in logging events.
+        start_time = (self.config.start_time - self.config.max_query_duration).strftime(
+            BQ_DATETIME_FORMAT
+        )
+        self.report.start_time = start_time
+
+        end_time = (self.config.end_time + self.config.max_query_duration).strftime(
+            BQ_DATETIME_FORMAT
+        )
+        self.report.end_time = end_time
```

Comment:
similar to line 403.

Reply:
will fix to rename these properly to avoid confusion